src/components/Search/SearchRouter/buildSubstitutionsMap.ts

```diff
@@ -0,0 +1,77 @@
+import type {OnyxCollection} from 'react-native-onyx';
+import type {SearchAutocompleteQueryRange, SearchFilterKey} from '@components/Search/types';
+import * as parser from '@libs/SearchParser/autocompleteParser';
+import * as SearchQueryUtils from '@libs/SearchQueryUtils';
+import CONST from '@src/CONST';
+import type * as OnyxTypes from '@src/types/onyx';
+import type {SubstitutionMap} from './getQueryWithSubstitutions';
+
+const getSubstitutionsKey = (filterKey: SearchFilterKey, value: string) => `${filterKey}:${value}`;
+
+/**
+ * Given a plaintext query and data
+ * this function will build from scratch
+ *
+ * Ex:
+ * query: `Test from:John1`
```

Comment:
Can we also add another example in which function returns a substitution? The comment seems misleading to me without it.

Reply:
ahh great catch! this example is a copy-paste error, I will update it